lib/aur-sync

```diff
@@ -333,11 +333,11 @@ fi
 cd_safe "$aur_workdir"
 
 if ((download)); then
-    xargs -a "$tmp"/queue aur fetch "${fetch_args[@]}"
+    parallel -X aur fetch "${fetch_args[@]}" :::: "$tmp"/queue
```

Comment:
I'm not sure why, but in the past I've seen that multiple jobs when cloning over SSH was problematic. Needs testing